package.json

```diff
@@ -39,7 +39,7 @@
   "scripts": {
     "lint": "eslint *.{js,mjs} src/**/*.{js,mjs} {test,dev}/*.{js,mjs} sample/*.{js,mjs}",
     "lint:fix": "eslint --fix *.{js,mjs} src/**/*.{js,mjs} {test,dev}/*.{js,mjs} sample*/*.{js,mjs}",
-    "build": "webpack && lessc style/crosswords.less dist/crosswords.css",
+    "build": "webpack && lessc style/crosswords.less dist/crosswords.css && git diff --exit-code",
```

Comment:
`git diff --exit-code` fails if there a working changes. Just a simple way to make sure the developer doesn't forget to build the `dist/` directory (which I had done!).

Reply:
Umm... I regularly process local changes as a series of commits, so having uncommitted changes is a common occurrence for me.

A better solution may be to amend our `pre-commit` Git hook to build the `dist/` directory?. This is documented in the root `README.md`

Reply:
Ok - I'll move it out.